app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -565,19 +565,21 @@ impl NodeModel {
         self.drag_area.size.set(padded_size);
         self.error_indicator.size.set(padded_size);
         self.vcs_indicator.set_size(padded_size);
-        self.backdrop.mod_position(|t| t.x = width / 2.0);
-        self.background.mod_position(|t| t.x = width / 2.0);
-        self.drag_area.mod_position(|t| t.x = width / 2.0);
-        self.error_indicator.set_position_x(width / 2.0);
-        self.vcs_indicator.set_position_x(width / 2.0);
+        let view_pos = view_position_of_node_with_size(size);
```

Comment:
The `backdrop`, `background`, `drag_area`, `error_indicator`, and `vcs_indicator` are all of `View` types, that's thus where I took the name idea from: https://github.com/enso-org/enso/blob/e9d031cb057c3a3b4a737c4645e788768a3475aa/app/gui/view/graph-editor/src/component/node.rs#L410-L413 . Would you have an idea of what other name could be better here? Would you think maybe `main_views_position_of_node_with_size`? Do we have some specific name for the `View` types that are the non-visualization parts of the Node?

The full name here as I currently understand it would be: "position of the View objects that are parts of a Node surrounding the expression text area, for a Node of given size". After some shortening, would a name `position_of_text_area_surrounding_object_of_node_with_size` be a better choice here? Or maybe, `text_area_position_of_node_with_size`?

Reply:
>  View types, that's thus where I took the name idea from: 

This makes sense! I was missing that context.

To be honest, I think all those things make up the node, so I would probably go simply with `position_of_node_with_size` instead.

Reply:
After further discussion on Discord, with @MichaelMauderer we currently agree on a name `background_offset` here.